src/components/pages/MyExpensesPage.js

```diff
@@ -1,100 +1,79 @@
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-    ];
-
-    const amount = '100€';
-    const percentage = '+10%';
-
-    return (
-        <Wrapper>
-            <Typography variant="h1">{t('myExpenses.title')}</Typography>            
-            <ContentContainer> 
+  const { t } = useTranslation();
+  const expenses = data.expenses;
+  const amount = '100€';
+  const percentage = '+10%';
 
-                <Grid container spacing={2}>
-                    <Grid item xs={6}>
-                        <StyledCard>
-                            <StyledButton variant="text">
-                                <Typography variant="h2">{t('myExpenses.addNew')}</Typography>
-                            </StyledButton>
-                        </StyledCard>
-                    </Grid>
+  return (
+    <Wrapper>
+      <Typography variant="h1">{t('myExpenses.title')}</Typography>
+      <ContentContainer>
+        <Grid container spacing={2}>
+          <Grid item xs={6}>
+            <StyledCard>
+              <StyledButton variant="text">
+                <Typography variant="h2">{t('myExpenses.addNew')}</Typography>
+              </StyledButton>
+            </StyledCard>
+          </Grid>
 
-                    <Grid item xs={6}>
-                        <StyledCard>
-                            <TotalBalanceItem amount={amount} percentage={percentage}/>
-                        </StyledCard>
-                    </Grid>
-                </Grid>
-        
-                <Grid container spacing={1}>
-                    <Grid item xs={12}>
-                        <StyledCard>
-                            <List>
-                                {expenses.map((expense, index) => (
-                                <ExpenseItem
-                                    key={index}
-                                    name={expense.name}
-                                    timestamp={expense.timestamp}
-                                    amount={expense.amount}
-                                />
-                                ))}
-                            </List>
-                        </StyledCard>
-                    </Grid>
-                </Grid>
+          <Grid item xs={6}>
+            <StyledCard>
+              <TotalBalanceItem amount={amount} percentage={percentage} />
+            </StyledCard>
+          </Grid>
+        </Grid>
 
-            </ContentContainer>
-      </Wrapper>
-    );
-  };
\ No newline at end of file
+        <Grid container spacing={1}>
+          <Grid item xs={12}>
+            <StyledCard>
+              <List>
+                {expenses.map((expense, index) => (
+                  <ExpenseItem
+                    key={index}
+                    title={expense.title}
```

Comment:
The description is not mapped from the data properly
```suggestion
                    title={expense.desc}
```
Also, we should think about the currencies because we should definitely show them. But do we want to allow a user to select one
- If yes, we need to add that info to the DB but it can cause some headaches with the calculation of the sum etc...
- If not, (I vote for that right now) it should be still shown by default `€`. It could still be a param though 

What do you think?

Reply:
I would NOT let user chose currencies as well. But in the data I would keep the amount just as a number = if we want to count the monthly reports, we will just sum it up so I will add the currency as a text to the component.

Reply:
yep, agreed